q2_quality_control/quality_control.py

```diff
@@ -32,21 +29,26 @@ def exclude_seqs(feature_sequences: DNAFASTAFormat,
     query_series = _dnafastaformats_to_series(feature_sequences)
 
     # if no hits are in res, early return empty hits and query_series as misses
-    if len(res) < 1:
+    if len(hit_ids) < 1:
         hits_seqs = pd.Series()
         return hits_seqs, query_series
     # if all query seqs are hits, return query_series as hits and empty misses
-    elif len(res) == len(query_series):
+    elif len(hit_ids) == len(query_series):
         misses_seqs = pd.Series()
         return query_series, misses_seqs
 
     # filter seqs from seq file
-    res_md = qiime2.Metadata(res)
-    hits_seqs = filter_seqs(query_series, res_md, exclude_ids=False)
-    misses_seqs = filter_seqs(query_series, res_md, exclude_ids=True)
+    hits_seqs = {}
```

Comment:
Could you put all of this in an `else` block that would follow the preceding `if`/`elif`? It shouldn't matter in practice, but it'd help with readability.

Reply:
👍